README.md

```diff
@@ -37,10 +37,13 @@ Want to read everything? Check out the [documentation](https://docs.dissect.tool
 
 Dissect currently consists of the following projects.
 
+- [dissect.archive](https://github.com/fox-it/dissect.archive)
 - [dissect.btrfs](https://github.com/fox-it/dissect.btrfs)
 - [dissect.cim](https://github.com/fox-it/dissect.cim)
 - [dissect.clfs](https://github.com/fox-it/dissect.clfs)
 - [dissect.cstruct](https://github.com/fox-it/dissect.cstruct)
+- [dissect.cobaltstrike](https://github.com/fox-it/dissect.cobaltstrike)
```

Comment:
```suggestion
```
This is currently viewed as a standalone project. It's also not installed by this package.